### DISCUSSION TITLE
MongoDB

### DISCUSSION THREAD

**Discussion by hakimLyon** [epicweb-dev/epic-stack#364] [Ideas]
Hi @kentcdodds , I hope you're in great shape! 
I have created an example of an epic stack application using MongoDB as the database. 
Can I make a PR for it?
[Epic Stack + MongoDB as the Database](https://github.com/hakimLyon/epic-stack-with-prisma-mongodb)

**Comment by kentcdodds** [comment]
Yeah! Make a PR for an example 👍 Thanks!

**Reply by hakimLyon** [reply]
Thanks 😊. I've already created PR. You can find it #365.

**Reply by kentcdodds** [reply]
Awesome. Thanks!
